test/Microsoft.ML.Tests/Transformers/TextFeaturizerTests.cs

```diff
@@ -197,5 +197,39 @@ public void NgramWorkout()
             CheckEquality("Text", "ngrams.tsv");
             Done();
         }
+
+        [Fact(Skip = "LdaNative dll is missing in ML.Net")]
```

Comment:
>Skip [](start = 14, length = 4)

test

Reply:
I will check once again. I do see LdaNative.dll in the build folder.

---
In reply to: [219620100](https://github.com/dotnet/machinelearning/pull/972#discussion_r219620100) [](ancestors = 219620100)